src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Shield.shared.cs

```diff
@@ -92,7 +160,7 @@ public Color TextColor
 		/// <summary>
 		/// Font size of all the text on the <see cref="Shield" />. <see cref="NamedSize" /> values can be used. This is a bindable preoprty.
 		/// </summary>
-		[TypeConverter(typeof(FontSizeConverter))]
+		[Xamarin.Forms.TypeConverter(typeof(FontSizeConverter))]
```

Comment:
Why is this needed?

Reply:
Because without it, there is a namespace ambiguity, (with  System.Runtime.CompilerServices;  if I remember well()